yarelismartin/SkillSwap
# Setup/ym

## Description
- Updatated the env. 
- Completed the to-do for this package. 
- Updated package.json with the name of the app and author
- Added the name of my app to the title tag

## Related Issue
#17 

## Motivation and Context
Essential to complete before starting project. 

## How Can This Be Tested?
When you pull branch you should be able to login to the app and see 'Skill Share' in the title of the tab. 
## Screenshots (if appropriate):

## Types of changes

[ ] Bug fix (non-breaking change which fixes an issue)
[x ] New feature (non-breaking change which adds functionality)
[ ] Breaking change
